apps/foundation-app/src/app/product/components/product-list/product-list.component.html

```diff
@@ -1,4 +1,11 @@
-<div class="product-container" *ngFor="let product of products">
-  {{product.id}}
-  {{product.cost}}
+<div class="card-container">
```

Comment:
This is probably two components, the `product-list` and the `product-card`. I'd probably add  another component to further separate. That way, once we add pagination, we can keep the concerns separate.

Reply:
Additionally, these class names are a bit imprecise. Luckily in Angular, we have ViewEncapsulation, but let's assume for some reason, we need to turn it off in the future. It would be painful if we happen to need a card/card-container class.

I think like `product-list` and `product-card` in a `BEM` style are good. http://getbem.com/naming/

Reply:
On top of that, I'd consider grabbing some sample data from https://github.com/magento/magento2-sample-data/tree/2.3-develop/pub/media to use for the images.